examples/deal.II/bp.cc

```diff
@@ -0,0 +1,235 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2023 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+//  Authors: Peter Munch, Martin Kronbichler
+//
+// ---------------------------------------------------------------------
+
```

Comment:
```suggestion

//TESTARGS(name="BP1") --resource {ceed_resource} --bp BP1 --fe_degree 2
//TESTARGS(name="BP4") --resource {ceed_resource} --bp BP4 --fe_degree 3

```

I think this should be a reasonable set of test cases? Our test harness would then run every libCEED backend against these two test cases once the top level Makefile is set up to correctly build this example

Reply:
Looks good to me!